pipelines/ApVerifyWithFakes.yaml

```diff
@@ -0,0 +1,9 @@
+description: Stub instrumented Alert Production pipeline
+#
+# This pipeline does not depend on the local ApVerify.yaml, because the definition
+# of the primary ApVerifyWithFakes.yaml is more likely to change than the
+# data-specific overrides, and importing both pipelines can't merge changes to
+# the same task.
```

Comment:
"...importing both pipelines can't merge changes to the same task." - I'm having trouble parsing this.

Reply:
If you try to import two pipelines (like `dataset/pipelines/ApVerify.yaml` and `ap_verify/pipelines/ApVerifyWithFakes.yaml`) and they both override the same task, Middleware won't let you do it (IIRC). You definitely won't get the same behavior as with multiple non-pipeline config overrides.